### PR TITLE
Refresh menu on media folder creation.

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by ghost** [comment]
@itkg-alavieille @itkg-nanne 
Cherry-pick of 1.0 branch modification into 1.1.

File: MediaAdminBundle/Resources/public/coffee/FolderFormView.coffee [hunk 1]
```diff
@@ -2,8 +2,10 @@ FolderFormView = OrchestraModalView.extend(
 
   onViewReady: ->
     if @options.submitted
-      displayMenu()
-
+      listUrl = appRouter.generateUrl('listFolder',
+        'folderId': $('#folder_id', @$el).val()
```
Comment (alavieille) [inline MediaAdminBundle/Resources/public/coffee/FolderFormView.coffee]: think to update the id (the name of form is changed)

**Comment by ghost** [comment]
@itkg-alavieille, id is now updated.

**Comment by alavieille** [comment]
:+1:

**Pull request merged**